config/locales/pl.yml

```diff
@@ -0,0 +1,164 @@
+      almost_x_years: "%{count} lat"
+      half_a_minute: Przed chwilą
+      less_than_x_minutes: "%{count}min"
+      less_than_x_seconds: Przed chwilą
+      over_x_years: "%{count} lat"
+      x_days: "%{count} dni"
+      x_minutes: "%{count}min"
+      x_months: "%{count} miesięcy"
+      x_seconds: "%{count}s"
+  exports:
+    blocks: Blokujesz
+    csv: CSV
+    follows: Śledzisz
+    storage: Media storage
+  generic:
+    changes_saved_msg: Ustawienia zapisane!
+    powered_by: uruchomione na %{link}
+    save_changes: Zapisz zmiany
+    validation_errors:
+      one: Coś jest wciąż nie tak! Przyjrzyj się błędowi poniżej
+      other: Coś jest wciąż nie tak! Przejrzyj błędy (%{count}) poniżej
+  imports:
+    preface: Możesz zaimportować pewne dane (jak dane kont, które śledzisz lub blokujesz) do swojego konta na tym serwerze, korzystająć z danych wyeksportowanych z innego serwera.
+    success: Twoje dane zostały załadowane i zostaną niebawem przetworzone
+    types:
+      blocking: Lista blokowanych
+      following: Lista śledzonych
+    upload: Załaduj
+  landing_strip_html: <strong>%{name}</strong> ma konto na <strong>%{domain}</strong>. Możesz je śledzić i wejść z nim w interakcję jeśli masz konto gdziekolwiek w Fediwersie. Jeśli jeszcze go nie masz, możesz <a href="%{sign_up_path}">stworzyć konto</a>.
+  notification_mailer:
+    digest:
+      body: 'Oto krótkie podsumowanie co Cię ominęło na %{instance} od Twojej ostatniej wizyty (%{since}):'
+      mention: "%{name} wspomniał Cię w:"
+      new_followers_summary:
+        one: Śledzi Cię nowa osoba! Gratulacje!
+        other: Kilka (%{count}) nowych osób Cię śledzi! Wspaniale!
+      subject:
+        one: "1 nowe powiadomienie od Twojej ostatniej wizyty \U0001F418"
+        other: "%{count} nowych powiadomień od Twojej ostatniej wizyty \U0001F418"
+    favourite:
+      body: 'Twój wpis został polubiony przez %{name}:'
+      subject: "%{name} lubi Twój wpis"
+    follow:
+      body: "%{name} Cię śledzi!"
+      subject: "%{name} Cię śledzi"
+    follow_request:
+      body: "%{name} poprosił o możliwość śledzenia Cię"
+      subject: 'Prośba o możliwość śledzenia: %{name}'
+    mention:
+      body: '%{name} wspomniał Cię w:'
+      subject: '%{name} Cię wspomniał'
+    reblog:
+      body: 'Twój wpis został podbity przez %{name}:'
+      subject: "Twój wpis został podbity przez %{name}"
+  pagination:
+    next: Następna
+    prev: Poprzednia
+  remote_follow:
+    acct: Podaj swój adres (nazwa@domena), z którego chcesz śledzić
+    missing_resource: Nie udało się znaleźć adresu przekierowania z Twojej domeny
+    proceed: Śledź
+    prompt: 'Śledzony będzie:'
+  settings:
+    authorized_apps: Uwierzytelnione aplikacje
+    back: POwrót do Mastodona
+    edit_profile: Edytuj profil
+    export: Exportuj dane
+    import: Importuj dane
+    preferences: Preferencje
+    settings: Ustawienia
+    two_factor_auth: Uwierzytelnianie dwustopniowe
```

Comment:
Wygląda na to, że tłumaczenie "dwustopniowe" też występuje: https://securingthehuman.sans.org/newsletters/ouch/issues/OUCH-201308_po.pdf